roofit/roofit/src/RooKeysPdf.cxx

```diff
@@ -20,7 +20,7 @@
 
 Class RooKeysPdf implements a one-dimensional kernel estimation p.d.f which model the distribution
 of an arbitrary input dataset as a superposition of Gaussian kernels, one for each data point,
-each contributing 1/N to the total integral of the p.d.f..
+each contributing 1/N to the total integral of the p.d.f.
```

Comment:
```suggestion
each contributing 1/N to the total integral of the pdf.
```
We prefer writing "pdf" without periods now, so you can change that while you are at it.